packages/vscode-extension/src/handlers.ts

```diff
@@ -847,18 +847,33 @@ export async function listCollaborator(env: string): Promise<TreeItem[]> {
       result = [
         {
           commandId: `fx-extension.listcollaborator.${env}`,
-          label: "No permission to list all collaborators.",
+          label: "No permission to list collaborators",
           icon: "warning",
           isCustom: true,
           parent: "fx-extension.environment." + env,
         },
       ];
     }
+    ExtTelemetry.sendTelemetryEvent(Stage.listCollaborator, {
```

Comment:
Also send start telemetry at the beginning？

Reply:
OK, thanks for the tip

Reply:
Updated